.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -58,8 +58,8 @@ steps:
 
   # We create the variable dry run and the update it, depending on the pipeline and branch type.
   - bash: |
-      export DRY_RUN=${{ parameters.dryrun }}
-      echo '##vso[task.setvariable variable=DRY_RUN]${DRY_RUN}'
+      DRY_RUN=${{ parameters.dryrun }}
+      echo "##vso[task.setvariable variable=DRY_RUN]${DRY_RUN}"
```

Comment:
This is the fix - to let the shell expand the $DRY_RUN variable.

Reply:
Is it possible to add an automated check like shell-check that detects such quote issues before a PR is merged?

Reply:
Maybe. I was about to google to find out if I can write unit tests for ADO pipelines. Will see if we can shell-check scripts embedded in the yaml too. 

I guess we really should extract some of this code into .sh files so it can be shell-checked.